tests/test_opds.py

```diff
@@ -92,7 +92,11 @@ def test_authors(self):
         edition.add_contributor(
             "Jonathan Frakes", Contributor.NARRATOR_ROLE
         )
-        author, contributor = Annotator.authors(None, edition)
+        author, contributor = sorted(
+            Annotator.authors(None, edition),
+            key=lambda x: x.tag
+        )
+        
```

Comment:
This change is unrelated. I got a spurious test failure because the tags come out of `authors` in a nondeterministic order.